gqlgen.yml

```
@@ -610,6 +610,8 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.APIResourceLimits
   ResourceLimitsInput:
     model: github.com/evergreen-ci/evergreen/rest/model.APIResourceLimits
+  RestartAdminTasksOptions:
+    model: github.com/evergreen-ci/evergreen/model.RestartOptions
```

You can do this to map `includeSystemFailed` to `includeSysFailed`
```suggestion
    model: github.com/evergreen-ci/evergreen/model.RestartOptions
    fields:
      includeSystemFailed:
        fieldName: includeSysFailed
```

---

ty!